servicetest/testframework/testhelper.py

```diff
@@ -180,10 +171,44 @@ def __file_name(self):
         return EnvironmentHelper.ENV_VARS_FILE_NAME
 
 
+class CGroupHelper(Helper):
+    """ Helper for examining Cgroups Gateway inside the container
+    """
+    def __init__(self, file_base_path):
+        Helper.__init__(self, file_base_path)
+
+    # Below methods are executed inside container
+    def allocate(self, size):
+        buffer = ""
+        step = size / 10
+        while (size > len(buffer)):
+            try:
+                buffer = buffer + ('a' * step)
+            except:
+                return False
```

Comment:
Is it possible to release the memory in this case or is it too late? If it is too late, it would be nice with a comment about that.